eng/PrepareRelease.yml

```diff
@@ -0,0 +1,43 @@
+stages:
+- stage: PrepareReleaseStage

```

Comment:
This stage seems to run unconditionally, so long as the publish_using_darc stage runs; but that stage always runs regardless of whether the build is published to a channel or not. I feel this should only be run on release branches, or at the very least only on official branches; it probably shouldn't be run on private branches.

Not a requirement (it's not a big deal if my private branch publishes some files to the blob storage account... maybe), but something to think about.

Reply:
I'm making this run only from the `release/*` branches.